searx/static/themes/simple/src/less/toolkit.less

```diff
@@ -494,6 +494,7 @@ select {
 
 /* -- engine-tooltip -- */
 .engine-tooltip {
+  right: 12rem;
```

Comment:
One last point on this PR.

This CSS is for all the tooltips. With `right: 12rem`, the tooltip over the engine name will be displayed on the right too.

So a possible solution:
* `.engine-tooltip` should be remain as before
* in this line: 
  https://github.com/searxng/searxng/blob/9ef7f38e4f20bad5a76867aa3e5049a92bc58e59/searx/templates/simple/preferences.html#L73
  add a new CSS class like `column-reliability`
* in searx/static/themes/simple/src/less/preferences.less , add:
```
.column-reliability {
  .engine-tooltip {
    right: 12rem;
  }
}
```

Reply:
fixed in bd830f7